pdf/core/utils.go

```diff
@@ -38,6 +38,18 @@ func getUniDocVersion() string {
 	return common.Version
 }
 
+// GetObjectNums returns the object numbers of the PDF objects in the file
+func (this *PdfParser) GetObjectNums() (map[int]int, error) {
```

Comment:
I use the object numbers to label objects in the output of my PDF object dumper program pdf_all_objects.go. When I am reading the output and looking through a decoded contentstream and see `22 0 R`, I want to be able to find object 22 in the output. 

I use a map because the `key` in `pdfReader.GetIndirectObjectByNumber(key)` is not the object number. The map is  `map[key] = objNum`.

A single function that iterates through all objects would be a better way of doing this.